web/filament-js/README.md

```diff
@@ -45,3 +45,10 @@ do `npm publish`.
 [yarn]: https://yarnpkg.com
 [unpkg]: https://unpkg.com
 [jsdelivr]: https://www.jsdelivr.com/
+
+9. Update the live drag-and-drop viewer as follows:
+
+   1. Edit the pinned Filament version in the `<script>` tag in `docs/viewer/index.html`.
+   2. Copy `web/filament-js/filament-viewer.js` to `docs/viewer`. To keep life simple, we do not
+      bother minifying this file.
```

Comment:
Strictly speaking, this step is only necessary if there was an API change that impacted the viewer.